CHANGELOG.md

```diff
@@ -7,10 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added support for including partial frames in `Watson.report` and use it in
+  `cli.aggregate` (#277)
```

Comment:
It looks like Watson's changelog _favors_ more high level comments, avoiding names of classes/methods, probably targeting Watson's users more than Watson's developers.

Reply:
I will change it the next time I get a few minutes for this.